kolibri/content/utils/channel_import.py

```diff
@@ -198,9 +212,67 @@ def generate_table_mapper(self, table_map=None):
         # If we got here, there is an invalid table mapping
         raise AttributeError('Table mapping specified but no valid method found')
 
-    def table_import(self, model, row_mapper, table_mapper, unflushed_rows):
```

Comment:
The `raw_attached_sqlite_table_import` method below is where most of the magic happens for the 10x metadata import speedup. It mounts the external content DB onto the main DB, and directly slurps things across in a single line of SQL.